src/components/AppWrapper/AppWrapper.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import type { DayPeriod } from '../../type/index.type';
+  export let period: DayPeriod;
+
+  const imageUrl = `url(images/${period}.png)`;
+</script>
+
+<div data-testid="wrapper-element" style={`--image: ${imageUrl}`} class="root">
```

Comment:
I think `wrapper` is enough, so `data-testid` will not be in the bundle after compiling?